src/components/SliderMenu.tsx

```diff
@@ -1,48 +1,100 @@
-import { Button, type DisplayTheme } from "@arconnect/components";
+import { type DisplayTheme } from "@arconnect/components";
 import { CloseIcon } from "@iconicicons/react";
+import { AnimatePresence, motion, type Variants } from "framer-motion";
+import type React from "react";
+import { useRef } from "react";
 import styled, { useTheme } from "styled-components";
-import { SendInput } from "~routes/popup/send";
 
 interface SliderMenuProps {
   title: string;
+  isOpen: boolean;
   onClose?: () => void;
   children?: React.ReactNode;
 }
 
 export default function SliderMenu({
-  children,
   title,
-  onClose
+  isOpen,
+  onClose,
+  children
 }: SliderMenuProps) {
+  const wrapperElementRef = useRef<HTMLDivElement | null>(null);
   const theme = useTheme();
-  return (
-    <Wrapper displayTheme={theme.displayTheme}>
-      <Body>
-        <Header>
-          <Title>{title}</Title>
-          <ExitButton onClick={onClose} />
-        </Header>
-        {children}
-      </Body>
-    </Wrapper>
-  );
+
+  const contentElement = isOpen ? (
+    <>
+      <CloseLayer
+        key="SliderMenuCloseLayer"
+        exit={{ opacity: 0 }}
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        transition={{ duration: 0.2 }}
+        onClick={(e) => {
+          e.stopPropagation();
+          onClose();
+        }}
+      />
+
+      <Wrapper
+        displayTheme={theme.displayTheme}
+        ref={wrapperElementRef}
+        variants={animationSlideFromBottom}
+        initial="hidden"
+        animate="shown"
+        exit="hidden"
+      >
+        <Body>
+          <Header>
+            <Title>{title}</Title>
+            <ExitButton onClick={onClose} />
+          </Header>
+          {children}
+        </Body>
+      </Wrapper>
+    </>
+  ) : null;
+
+  return <AnimatePresence>{contentElement}</AnimatePresence>;
 }
 
 const ExitButton = styled(CloseIcon)`
   cursor: pointer;
 `;
 
-const Wrapper = styled.div<{ displayTheme: DisplayTheme }>`
+const Wrapper = styled(motion.div)<{ displayTheme: DisplayTheme }>`
   position: fixed;
+  bottom: 0;
+  left: 0;
+  height: auto;
+  max-height: calc(2 / 3 * 100%);
   display: flex;
   flex-direction: column;
-  min-height: 100%;
-  padding-bottom: 62px;
+  padding-bottom: 16px;
   width: 100%;
+  z-index: 1000;
+  overflow: scroll;
   background-color: ${(props) =>
     props.displayTheme === "light" ? "#ffffff" : "#191919"};
+  border-radius: 10px 10px 0 0;
```

Comment:
The white background mixed with the white in the CloseLayer makes this really hard to see in Light mode
Wdyt about maybe adding a subtle shadow or border to make it easier to see the slider covering the CloseLater/background items?

Reply:
moved the slider to go higher which shows less of the components behind it, we can add a border if it's still too distracting